src/models.py

```diff
@@ -13,12 +14,16 @@
 
 
 class CoreModelMixin(BaseModel, metaclass=TableMeta):
+    __repr_fields__: List[str] = []
```

Comment:
It might be worth looking into the dataclass integration to see if that will simplify the need to create things like this

Reply:
I will do this in a separate issue